contracts/utils/EnumerableSet.sol

```diff
@@ -20,58 +20,60 @@ pragma solidity ^0.6.0;
 library EnumerableSet {
 
     struct AddressSet {
-        // Position of the value in the `values` array, plus 1 because index 0
-        // means a value is not in the set.
-        mapping (address => uint256) index;
-        address[] values;
+        address[] keys;
+        // Position of the key in the `keys` array, plus 1 because index 0
+        // means a key is not in the set.
+        mapping (address => uint256) indexes;
```

Comment:
Why did you reorder the members of the struct? I'm weary of reordering because of the effect on upgradeable contracts.

Reply:
I thought it made the code more readable :man_shrugging: And 3.0 will not be upgrade-compatible with 2.0 anyway, so I thought this was a good time to do it.

Reply:
Is there a list of storage-breaking changes? I still feel that we should avoid them if possible. But I'm unsure.

Reply:
We don't, no.

Is there a reason to avoid them other than allowing for upgrades to happen? Given the extent of the API changes from 2.0 to 3.0 I'd discourage people from doing so.

Reply:
Only upgrades.